.woodpecker/ci.yml

```diff
@@ -9,11 +9,16 @@ clone:
             depth: 10
             lfs: false
 
+matrix:
+    CONTAINER_IMAGE:
+        - registry.gitlab.com/fun-tech/fundraising-frontend-docker:ci
+        - registry.gitlab.com/fun-tech/fundraising-frontend-docker:php-8.4
```

Comment:
I think this needs to be `registry.gitlab.com/fun-tech/fundraising-frontend-docker/php-8.4:ci`